modules/tinymce/CHANGELOG.md

```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 - Text alignment could not be applied to `pre` elements #TINY-7715
 - UI components, such as dialogs, would in some cases cause the `esc` keyup event to incorrectly trigger inside the editor #TINY-7005
 - Selection direction was not stored/restored when getting/setting selection bookmarks #TINY-8599
+- Selection could cause an error on some versions of Safari #TINY-8686
```

Comment:
```suggestion
- An exception could be thrown for the `editor.selection.isForward()` API due to an invalid selection on some versions of Safari #TINY-8686
```
Maybe? Either way we probably need to be clearer here as "Selection" isn't clear in this case.